wremnants/CardTool.py

```diff
@@ -290,7 +290,7 @@ def addLnNSystematic(self, name, size, processes, group=None, groupFilter=None,
     # use doActionBeforeMirror to do something before it instead (so the mirroring will act on the modified histogram)
     # decorrelateByBin is to customize eta-pt decorrelation: pass dictionary with {axisName: [bin edges]}
     def addSystematic(self, name, systAxes=[], systAxesFlow=[], outNames=None, skipEntries=None, labelsByAxis=None, 
-                      baseName="", mirror=False, mirrorDownVarEqualToUp=False, mirrorDownVarEqualToNomi=False,
+                      baseName="", mirror=False, mirrorDownVarEqualToUp=False, mirrorDownVarEqualToNomi=False, symmetrize = True, symmetrizeConservative = False,
```

Comment:
Should probably just be one argument that takes e.g., None, 'SymmetrizeConservative', 'symmetrize'

Reply:
Yah ok can change that.